src/routes/Project/actions/TrackActionHandler.js

```diff
@@ -47,6 +48,16 @@ const ACTION_HANDLERS = {
     let isMovingRight = oldClip.startTime < newClip.startTime;
     let isMovingLeft = !isMovingRight;
 
+
+    // grid snapping
```

Comment:
that doesnt make sense. x % 1 is always 0 <= result < 1, so remainder will always be less than the grid snap threshold